build/yamls/antrea-aks.yml

```diff
@@ -680,8 +844,9 @@ data:
     #  Traceflow: false
 
     # Enable ClusterNetworkPolicy feature to complement K8s NetworkPolicy for cluster admins
-    # to define security policies which apply to the entire cluster.
-    #  ClusterNetworkPolicy: false
+    # to define security policies which apply to the entire cluster, and AntreaNetworkPolicy
```

Comment:
should it have a space Antrea and NetworkPolicy since it's no longer the name of the kind?

Reply:
It name of the kind has always been NetworkPolicy... just that it is under security/v1alpha1 as opposed to the K8s networking/v1

Reply:
I think Quan means to change AntreaNetworkPolicy to "Antrea NetworkPolicy" in the comments, which I would agree.
Maybe change "ClusterNetworkPolicy" to "Antrea ClusterNetworkPolicy" as well.

Reply:
Done